docker-compose.yml

```diff
@@ -7,21 +7,21 @@ services:
     ports:
       - 5000:5000
     environment:
-      - DATABASE_URL=postgres://postgres:postgres@db:5432/web
-      - SECRET_KEY=09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7
-      #used to ping the database to check if it is up before connecting
-      - POSTGRES_USER=postgres
-      - POSTGRES_PASSWORD=postgres
-      - POSTGRES_DB=web
+      - DATABASE_URL=${DATABASE_URL}
+      - SECRET_KEY=${SECRET_KEY}
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+
     volumes:
       - ./services/backend:/app
     #command: bash -c "aerich upgrade && uvicorn src.main:app --reload --host 0.0.0.0 --port 5000"
   
     command: >
-      bash -c "./wait_for_postgres.sh db 'aerich upgrade && uvicorn src.main:app --reload --host 0.0.0.0 --port 5000'"
+      bash -c "./wait_for_postgres.sh srcbot1.c7okouiqu9jh.us-east-1.rds.amazonaws.com 'aerich upgrade && uvicorn src.main:app --reload --host 0.0.0.0 --port 5000'"
```

Comment:
Why are we calling ./wait_for_postgres script when the RDS database is already online?